src/Webapi/Dom/Webapi__Dom__RectList.res

```diff
@@ -0,0 +1,9 @@
+type t = {
+  length: int
+}
+
+@val external toArray: t => array<Dom.domRect> = "Array.prototype.slice.call"
+
+@get external length: t => int = ""
```

Comment:
Should this be = "length"?

Reply:
whoops, I didn't build this on top of the explicit externals branch so I forgot about these.